package.json

```diff
@@ -57,10 +57,8 @@
   ],
   "license": "BSD-3-Clause",
   "lint-staged": {
-    "./*.js": "npm run lint-arg --",
     ".eslintignore": "npm run lint",
-    "src/**/*.js": "npm run lint-arg --",
-    "test/**/*.js": "npm run lint-arg --"
+    "*.js": "eslint --report-unused-disable-directives --fix"
```

Comment:
As mentioned, I'd like to keep `lint-arg` so that we have all of our lint commands together, better ensuring they remain in sync.

Reply:
We do have `lint-fix`. I'm not always a fan of auto-fixing, but if you want to switch to `lint-fix`, I'm ok with that.

Reply:
Probably good in a group project as some folks making minor fixes might get frustrated and not know what to do with some linting directives.

Reply:
`lint-fix` for this command doesn't make sense, because `lint-staged` individually passes in the files, rather than calling the global lint job. I did go back to using your `lint-arg` consistently across all the calls though

Reply:
> `lint-fix` for this command doesn't make sense, because `lint-staged` individually passes in the files, rather than calling the global lint job. 

Sure, but we could try adding some `lint-*` script for the sake of `.eslintignore` which did run things globally with a fix. It appears `lint-staged` nowadays auto-runs `git add .` so maybe it would work? Anyways, it's not really that important here.

> I did go back to using your `lint-arg` consistently across all the calls though

Cool. I also really like what you did with `lint-fix` and `lint`.

Reply:
Do you want me to try and add the `lint-fix` in this PR, or keep it for later?

Reply:
I can merge what you have and see if it works. `lint-staged` warns against using `git add`, but will see if it can work.